sdk/appcenter-distribute/src/main/res/values-cs/appcenter_distribute.xml

```diff
@@ -1,21 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <string name="appcenter_distribute_update_dialog_title">K dispozici je aktualizace aplikace.</string>
+  <string name="appcenter_distribute_update_dialog_title">K dispozici je aktualizace aplikace</string>
   <string name="appcenter_distribute_update_dialog_message_optional">%1$s %2$s (%3$d) je k dispozici ke stažení a instalaci.</string>
   <string name="appcenter_distribute_update_dialog_message_mandatory">Vývojáři této aplikace vyžadují, abyste si nainstalovali aktualizaci %1$s %2$s (%3$d).</string>
   <string name="appcenter_distribute_update_dialog_download">Stáhnout</string>
   <string name="appcenter_distribute_update_dialog_postpone">Zeptat se za den</string>
   <string name="appcenter_distribute_update_dialog_view_release_notes">Zobrazit zprávu k vydání verze</string>
-  <string name="appcenter_distribute_dialog_actioned_on_disabled_toast">Distribuce byla zakázána.</string>
+  <string name="appcenter_distribute_dialog_actioned_on_disabled_toast">Distribuce byla zakázána</string>
   <string name="appcenter_distribute_unknown_sources_dialog_message">Z bezpečnostních důvodů je vaše zařízení nastaveno tak, aby blokovalo instalaci aplikací získaných z neznámých zdrojů.</string>
   <string name="appcenter_distribute_unknown_sources_dialog_settings">Nastavení</string>
-  <string name="appcenter_distribute_downloading_mandatory_update">Stahuje se aktualizace.</string>
+  <string name="appcenter_distribute_downloading_mandatory_update">Stahuje se aktualizace</string>
   <string name="appcenter_distribute_download_progress_number_format">%1$d MB z %2$d MB</string>
-  <string name="appcenter_distribute_install_ready_title">Aktualizace aplikace se stáhla.</string>
+  <string name="appcenter_distribute_install_ready_title">Aktualizace aplikace se stáhla</string>
```

Comment:
I don't understand why/how you localized if not from touch down, that should be a separate step.

Reply:
As mentioned, it should be done after this one. We need to run TouchDown for this string.

Reply:
That does not answer my question ;) Why did you localize yourself then?

Reply:
Well, just in case if we forgot getting localized text when we release newer version.

Reply:
But how can we make sure these translations are suitable? Where do they come from? Shouldn't we always commit english only and then ask TD in a separate step?

Reply:
As told offline, I had extra time to do this. 😄 Will use English for future changes.